code/Model/Recipient/EmailRecipient.php

```diff
@@ -534,7 +534,7 @@ public function canSend($data, $form)
     public function emailTemplateExists($template = '')
     {
         $t = ($template ? $template : $this->EmailTemplate);
-
+        $t = $t ?? '';
```

Comment:
```suggestion
        $t = ($template ? $template : $this->EmailTemplate) ?? '';
```
No need to do a second assignment.

The alternative would be to instead change the line below:

```diff
- return array_key_exists($t, (array) $this->getEmailTemplateDropdownValues());
+ return array_key_exists($t ?? '', (array) $this->getEmailTemplateDropdownValues());
```

Reply:
Updated